### DISCUSSION TITLE
I use wezterm with kitty keyboard protocol, why ctrl+i still a tab

### DISCUSSION THREAD

**Discussion by Abdillah** [nushell/nushell#9729] [Q&A] [answered]
Hello,

I want to set nushell vi motion with Colemak keyboard. It is `hnei` instead of `hjkl`. In adjacent to that, I want to capture <kbd>ctrl</kbd>+<kbd>i</kbd> to deal with history hint. Unfortunately, when I listen the key, it is actually still assumed as tab despite I have use kitty keyboard protocol in wezterm config.

How to enable kitty in nushell as well? Or currently it doesn't support it.

I found that crossterm has `KeyboardEnhancementFlags::DISAMBIGUATE_ESCAPE_CODES` but nowhere to be found in reedline code.

**Comment by amtoine** [comment]
@Abdillah 
from Reedline, i'm not sure the Vi-motion bindings are configurable right now :thinking:

**Reply by Abdillah** [reply]
The motion itself is configurable actually. It just not documented well.

**Reply by Abdillah** [reply]
Oh, do you mean the overall vi-motion? I just looked the source and it actually hard-coded in the Vi command parser.
What I understood was the `hjkl` only..

**Comment by amtoine** [comment]
is this a duplicate of https://github.com/nushell/nushell/issues/9733?

**Reply by Abdillah** [reply]
I'm sorry. I created that the next hour. I guess it actually is not implemented. I created a PR as well, hope everything goes well.

**Reply by amtoine** [reply]
no worries :+1: 

it's just that, if there is an issue and a PR for this already, i think we can close this discussion :wink:

**Comment by Abdillah** [comment, marked as answer]
It is not implemented indeed. The implementation is on going on the issue and PR linked.
Issue: https://github.com/nushell/nushell/issues/9733

**Reply by amtoine** [reply]
thanks for closing this, looking forward to the fix :blush:
